src/dialogs/search-dialog.js

```diff
@@ -119,7 +119,7 @@ class SearchDialog extends PromptDialog {
     const facet = missingEntities.keys().next().value;
     const facetValueCounts = (await this.db.getValuesByFacet([facet], this.query))[facet];
     return {
-      facetValueCounts,
+      facetValueCounts
```

Comment:
does this pass the style checking?
usually we put a comma

Reply:
How do I run style checking? Will revert to previous version.

Reply:
`yarn run style`

Could you add CI with tests and linting? (see https://github.com/Botfuel/botfuel-dialog/blob/master/.circleci/config.yml, I can help you if you want)

Reply:
I don't think there are any style tests... Yes, I can do that, and will let you know if I need help.

Reply:
There are linting and tests: 
```
      - run: yarn run style
      - run: yarn test
```